README.md

```diff
@@ -200,13 +200,29 @@ Then you can simply point the converted model path in the config file by changin
 
 For further information, please refer to [#15](https://github.com/facebookresearch/maskrcnn-benchmark/issues/15).
 
+To finetune the pretrained maskRCNN to train fasterRCNN or maskRCNN on your own dataset, four steps are needed: <br />
+
+**step 1**. Run the demo.py to test your package is installed successfully. Meanwhile, you can save the pretrained model, which has been converted from pkl format to pth format, which can be loaded by a pytorch model. The model is just a data of COCODEmo object in the demo. This way, no need to look into [_load_file fun](https://github.com/wmmxk/maskrcnn-benchmark/blob/f25c6cff92d32d92abe8965d68401004e90c8bee/maskrcnn_benchmark/utils/checkpoint.py#L117).
+
+**step 2**. Create a fastercnn or maskRCNN model using a yaml file in the configs folder. When doing this, you need to call the cfg.merge_from_list method to change the [_C.MODEL.ROI_BOX_HEAD.NUM_CLASSES =81 ](https://github.com/wmmxk/maskrcnn-benchmark/blob/f25c6cff92d32d92abe8965d68401004e90c8bee/maskrcnn_benchmark/config/defaults.py#L182) to the number of classes in your dataset (background does not count in this implementation).
+
+**step 3**. Create a customized dataset class that generates a tuple (img, bbox, label) each time. img is generated by Image.open(...).convert('RGB'); bbox is a 2d numpy array and label is 1d array. Then you can create another class on top of the class. For this inheritance step, it will be similar to this method [__getitem__](https://github.com/wmmxk/maskrcnn-benchmark/blob/f25c6cff92d32d92abe8965d68401004e90c8bee/maskrcnn_benchmark/data/datasets/coco.py#L35)
```

Comment:
The dataset should actually return `(img, target, idx)`, where `target` is a `BoxList` (which can contain the labels inside), see [this part](https://github.com/wmmxk/maskrcnn-benchmark/blob/f25c6cff92d32d92abe8965d68401004e90c8bee/maskrcnn_benchmark/data/datasets/coco.py#L43-L48) for more information, and `idx` is the index of the image (exactly what is passed to the `__getitem__`.